README.md

```diff
@@ -19,11 +19,13 @@ list of operating system distributions, package names, and available versions.
 Depending on the package, you may still
 need to source it from either `/etc/bashrc` or `~/.bashrc` (or any
 other file sourcing those). If you have _only_ bash >= 4.2 installed, you can
-do this by simply using:
+do this by using:
 
-```shell
-# Use bash-completion, if available
-[[ $PS1 && -f /usr/share/bash-completion/bash_completion ]] && \
+```bash
+# Use bash-completion, if available, and avoid double-sourcing
+[[ $PS1 &&
+  -z ${BASH_COMPLETION_VERSINFO:-} &&
```

Comment:
This doesn't follow our coding style. The coding style is meant for the actual codebase for `bash-completion`, but I think there's no reason to use other styles in documents.

```suggestion
  ! ${BASH_COMPLETION_VERSINFO:-} &&
```